### ISSUE TITLE
Importer for current enrolments - Feature request

### ISSUE THREAD

**Issue by reidka** [uoft-tapp/old-orig-tapp#177]
I'd like to be able to import/update the current enrolments for a course.  

Below is the format of the data that I can get from the FAS registrar's office.  Note that it includes both fall and spring terms, Lecture and Tutorials.  The full file includes courses from all disciplines.

Here is the format (fixed width fields):
```
Term  Department CourseCode Title Section Type  CCCCEEEE000000WWWW
```
- Term - this is related to session for us - The importer may need an equivalence, or a configurable parameter to say which session is relevant in the import data.
- Department - We only want CSC
- Course Code - this will mostly correspond to our positions but may be a substring of our positions (e.g. CSC108H1-A - (Probably this means that I should be more careful in specifying courses)
- Title - We can ignore this.
- Section - I'd like the sum of all the LEC sections to be reported for each course.  (Not TUT)
- Type - this can be ignored
- CCCC - enrolment cap
- EEEE - current enrolment
- 0s and mysterious unicode characters - not sure why these are here
- WWWW - wait list 

In the Summary view, I'd like to see "Cap" "Enrolment" "Waitlist"

In the ABC view, I'd like to see the current enrolment  in the title bar of the course beside the number of positions filled.

Special cases:
- we might have multiple different positions for a single course.  I'd like the total enrolment reported for each position.  
- we might have courses in our app that are not represented in the data like the HLP positions.  Just report enrolments and caps.

```
20179 CSC  CSC108H1F Intro to Comp Prog            LEC0101  YP  01960196�0000000150
20179 CSC  CSC108H1F Intro to Comp Prog            LEC0201  YP  01960196�0000000142
20179 CSC  CSC108H1F Intro to Comp Prog            LEC0301  YP  01960196�0000000141
20179 CSC  CSC108H1F Intro to Comp Prog            LEC0401  YP  01960196�0000000144
20179 CSC  CSC108H1F Intro to Comp Prog            LEC5101  YP  01960196�0000000162
20179 CSC  CSC108H1F Intro to Comp Prog            LEC9901  YP  01960196�0000000152
20181 CSC  CSC108H1S Intro to Comp Prog            LEC0101  YP  01960196�0000000031
20181 CSC  CSC108H1S Intro to Comp Prog            LEC0201  YP  00600060�0000000022
20181 CSC  CSC108H1S Intro to Comp Prog            LEC0301  YP  01960196�0000000018
20181 CSC  CSC108H1S Intro to Comp Prog            LEC5101  YP  01960196�0000000040
20181 CSC  CSC108H1S Intro to Comp Prog            LEC9901  YP  01960196�0000000024
20181 CSC  CSC108H1S Intro to Comp Prog            TUT0201  NE  00000000�0000000000
20181 CSC  CSC108H1S Intro to Comp Prog            TUT0202  NE  00000000�0000000000
20181 CSC  CSC108H1S Intro to Comp Prog            TUT0203  NE  00000000�0000000000
20179 CSC  CSC120H1F CompSci for Sciences          LEC0101  Y   01960185�0000000000
20181 CSC  CSC121H1S Comp Sci for Stats            LEC0101  Y   01960196�0000000052
20179 CSC  CSC148H1F Intro to Comp Sci             LEC0101  YP  02400240�0000000065
```

**Comment by michellemtchai** [comment]
@reidka The widths are exactly the same in any instance of this enrolment file right? Also, what file extension should I name this sort of file?

**Comment by gabriellesc** [comment]
The current enrollment for individual courses will be updateable through the courses view (it will replace "Estimated enrollment"). Would you also like to be able to update the cap and/or the waitlist through the courses view?

**Comment by gabriellesc** [comment]
Would you like an import button for this functionality (through the summary view)?

**Comment by reidka** [comment]
Yes.

**Comment by gabriellesc** [comment]
Yes to me? To both questions?

**Comment by reidka** [comment]
Sorry.  I was trying to answer without enough time. :-(  

Yes, I'd like to update cap, waitlist and current enrolments through the courses view.  Cap doesn't change often but it can change.

Yes to the button in the summary view to upload the enrolments data

**Comment by gabriellesc** [comment]
This should be implemented in `master` subsequent to #181.
